charts/pulsar/templates/proxy/proxy-cluster.yaml

```diff
@@ -0,0 +1,230 @@
+  imagePullPolicy: {{ .Values.images.proxy.pullPolicy }}
+  pod:
+    labels:
+      {{- include "pulsar.template.labels" . | nindent 6 }}
+      component: {{ .Values.proxy.component }}
+    annotations:
+      {{- if .Values.monitoring.datadog }}
+      {{- include "pulsar.proxy.datadog.annotation" . | nindent 6 }}
+      {{- end }}
+      prometheus.io/scrape: "true"
+      prometheus.io/port: "{{ .Values.proxy.ports.http }}"
+{{- with .Values.proxy.annotations }}
+{{ toYaml . | indent 6 }}
+{{- end }}
+    {{- if and .Values.affinity.anti_affinity .Values.proxy.affinity.anti_affinity }}
+    affinity:
+      podAntiAffinity:
+        {{ .Values.proxy.affinity.type }}:
+          {{ if eq .Values.proxy.affinity.type "requiredDuringSchedulingIgnoredDuringExecution" }}
+          - labelSelector:
+              matchExpressions:
+                - key: "app"
+                  operator: In
+                  values:
+                    - "{{ template "pulsar.name" . }}"
+                - key: "release"
+                  operator: In
+                  values:
+                    - {{ .Release.Name }}
+                - key: "component"
+                  operator: In
+                  values:
+                    - {{ .Values.proxy.component }}
+            topologyKey: "kubernetes.io/hostname"
+          {{ else }}
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: "app"
+                    operator: In
+                    values:
+                      - "{{ template "pulsar.name" . }}"
+                  - key: "release"
+                    operator: In
+                    values:
+                      - {{ .Release.Name }}
+                  - key: "component"
+                    operator: In
+                    values:
+                      - {{ .Values.proxy.component }}
+              topologyKey: "kubernetes.io/hostname"
+          {{ end }}
+    {{- end }}
+    {{- if .Values.proxy.resources }}
+    resources:
+      requests:
+        memory: "{{ .Values.proxy.resources.requests.memory }}"
+        cpu: "{{ .Values.proxy.resources.requests.cpu }}"
+    {{- end }}
+    {{- if .Values.proxy.securityContext }}
+    securityContext: {{ .Values.proxy.securityContext }}
+    {{- end }}
+    {{- if .Values.proxy.nodeSelector }}
+    nodeSelector:
+{{ toYaml .Values.proxy.nodeSelector | indent 6 }}
+    {{- end }}
+    {{- if .Values.proxy.tolerations }}
+    tolerations:
+{{ toYaml .Values.proxy.tolerations | indent 6 }}
+    {{- end }}
+    terminationGracePeriodSeconds: {{ .Values.proxy.gracePeriod }}
+    jvmOptions:
+      extraOptions: []
+      memoryOptions:
+        - {{ .Values.proxy.configData.PULSAR_MEM }}
+      gcOptions:
+        - {{ .Values.proxy.configData.PULSAR_GC }}
+      gcLoggingOptions: []
+  config:
+    tls:
+      enabled: {{ and (.Values.tls.enabled) (.Values.tls.proxy.enabled) }}
+    custom:
```

Comment:
We may need to put custom configs in `configData` here?

Reply:
.Values.zookeeper.configData is already